src/pages/Home.tsx

```diff
@@ -1,13 +1,113 @@
 import { tsx } from '@dojo/framework/widget-core/tsx';
+const hero = require('../assets/herobg.png');
 
 import * as css from './Home.m.css';
-
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import Card from '../widgets/card/Card';
+import Intersection from '@dojo/framework/widget-core/meta/Intersection';
+import has from '@dojo/framework/has/has';
 export default class Home extends WidgetBase {
 	protected render() {
+
+		const { isIntersecting } = this.meta(Intersection).get('cli');
+
+		const play = isIntersecting || has('build-time-render');
+
 		return (
-			<div>
-				<h1 classes={[css.root]}>Home</h1>
+			<div classes={[css.root]}>
+				<div styles={{ backgroundImage: `url(${hero})` }} classes={[css.hero]}>
+					<h1 classes={[css.headline]}>A Progressive Framework for Modern Web Apps</h1>
+					<button classes={[css.build]}>Build with Dojo</button>
+				</div>
+				<div classes={[css.ethos]}>
+					<Card extraClasses={{ root: css.ethosPoint }}>
+						<div classes={[css.ethosTitleContainer]}>
+							<span classes={css.productive}>
+								<FontAwesomeIcon icon="code-branch" />
+							</span>
+							<h3 classes={[css.ethosTitle]}>Productive</h3>
+						</div>
+						<div>
+							Dojo enables teams to build web applications with a deliberate approach to productivity,
+							sustainability and code management.
+						</div>
+					</Card>
+					<Card extraClasses={{ root: css.ethosPoint }}>
+						<div classes={[css.ethosTitleContainer]}>
+							<span classes={css.adaptable}>
+								<FontAwesomeIcon icon="plug" />
+							</span>
+							<h3 classes={[css.ethosTitle]}>Adaptable</h3>
+						</div>
+						<div>
+							Intent on not reinventing the wheel, Dojo allows for easy integration with the most powerful
+							solutions available today on the open web.
+						</div>
+					</Card>
+					<Card extraClasses={{ root: css.ethosPoint }}>
+						<div classes={[css.ethosTitleContainer]}>
+							<span classes={css.inclusive}>
+								<FontAwesomeIcon icon="users" />
+							</span>
+							<h3 classes={[css.ethosTitle]}>Inclusive</h3>
+						</div>
+						<div>
+							Demand for accessibility and internationalization are required for enterprise web
+							applications. Dojo supports inclusivity and provides both.
+						</div>
+					</Card>
+				</div>
+				<div classes={[css.getGoing]}>
+					<h2>Get Going Quickly</h2>
+					<p classes={[css.getGoingHeadline]}>
+						Getting started with Dojo is simple. You can use your command line of choice and{' '}
+						<a href="https://www.npmjs.com">npm</a> to get going quickly.{' '}
+					</p>
+					<div key="cli" classes={[css.cli]}>
+						<Card dark={true} extraClasses={{ root: css.commands }}>
+							<div classes={css.command}>
+								<span classes={[css.commandOne, play ? css.commandOneAnimation : null]}>npm i @dojo/cli @dojo/cli-create-app -g</span>
+								<span classes={[play ? css.blinkOne : null]}>|</span>
+							</div>
+							<div classes={css.command}>
```

Comment:
Are we going to be consistent with using arrays with vnode class props?

Reply:
Yes good shout. We probably need to do this everywhere at some point.